packages/drizzle/src/updateMany.ts

```diff
@@ -55,12 +56,21 @@ export const updateMany: UpdateMany = async function updateMany(
 
     const table = this.tables[tableName]
 
-    const docsToUpdate = await _db
-      .select({
-        id: table.id,
-      })
-      .from(table)
-      .where(where)
+    const docsToUpdate =
+      typeof limit === 'number'
```

Comment:
I think we should sanitize the limit before it is used. 
In postgres limit 0 is infinite whereas SQLite uses -1. In either case I would let it go through as no limit.
This is important to any core logic we introduce or plugins that might try to use the db adapter methods directly and will result in SQLite updating 0 rows.

Reply:
Done! Also added 2 tests